README.md

```diff
@@ -61,8 +60,28 @@ Note: If you have cloned the repo previously make sure you remove your node_modu
 - `./gradlew publishAndroid`
 - The compiled version of the Android module is here: `<project-root>/android`
 
+Note: On Windows the RealmJS repo should be cloned with symlynks enabled 
+```
+#run in elevated command prompt
+git clone -c core.symlinks=true https://github.com/realm/realm-js
+
+```
+
+or manually create the symlinks using directory junctions
```

Comment:
When would one want to do it manually instead of like above?

Reply:
when one has the repo already cloned